react/components/atoms/container-openable/container-openable.js

```diff
@@ -221,7 +223,6 @@ export class ContainerOpenable extends PureComponent {
                             onLayout={event => this._onHeaderLayout(event)}
                             {...this.props.headerProps}
                         >
-                            <View style={styles.knob} />

```

Comment:
Won't this cause regressions? Shouldn't it be optional or something?

Reply:
I thought about a prop for this, but also in my opinion if it is not pressable, why have a knob. @BeeMargarida @beneditomauro  @3rdvision  do you know of any use cases of this component where the knob is intended to show but it is not meant to be pressable/openable by touch?

Reply:
It's used in the Robin chat I believe.

Reply:
Yes, I don't think this can be removed

Reply:
can't be removed:
<img width="405" alt="image" src="https://user-images.githubusercontent.com/25725586/124913364-f02ec080-dfe6-11eb-9c4e-ce0742baf694.png">